src/app/core/config/config-fix.ts

```diff
@@ -712,8 +712,16 @@ export const defaultJsonConfig = {
           "components": [
             {
               "title": "",
-              "component": "EducationalMaterial"
-            }
+              "component": "EducationalMaterial",
+              "config": {
+               "summaries": [
+                {
+                  total: true,
+                  average: true,
+                }
+               ]
```

Comment:
```suggestion
               "summaries": {
                 total: true,
                 average: true,
               }
```
this would be even more straight forward, making it `@Input() summaries: {total: boolean; average: boolean};`

(still needs another small update of the component code of course ... Could you adapt that yet again? :blush:

Reply:
@sleidig, updated it.